docs/care/CEP/Draft/0009-pharmacy.md

```diff
@@ -0,0 +1,102 @@
+# CEP 9: Pharmacy Management
+
+## Motive
+
+CARE already includes prescription management and doctor prescription creation. This enhancement aims to extend the existing functionality to include robust pharmacy-specific features, improving medication dispensing, inventory control, and overall pharmacy operations.
+
+## Requirements
+
+### 1. Store Management
+
+- Support multiple main stores (warehouses) and multiple substores (stocking points) within a facility
```

Comment:
would need multiple layers of substores and a single MAIN store for all inlet of all itineraries